styles/stylesheet.css

```diff
@@ -0,0 +1,555 @@
+.header-container {
+  background-image:
+    url("/Users/Admin/Desktop/html200/html200-adopt-a-dog/images/header-image.jpg"),
+    radial-gradient(circle, #1261A0 90%, white);
+  background-repeat: no-repeat;
+  display: grid;
+  justify-content: space-around;
+  width: auto;
+  height: auto;
+}
+
+@media screen and (max-width: 576px) {
+  body {
+    display: grid;
+    grid-template-columns: 1fr;
+    padding: 1em;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  body {
+    display: grid;
+    grid-template-columns: 1fr;
```

Comment:
I think you may be trying to change the grid layout here but you are targeting the wrong element. dog-section-container-front should be changing grid settings.